### PR TITLE
New version: Hesan.He3 version 1.3.7

### PR DESCRIPTION
**Pull request by SpecterShell**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/106229)

### CODE REVIEW

**Comment by stephengillie** [comment]
image

File: manifests/h/Hesan/He3/1.3.7/Hesan.He3.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.2.4 $debug=QUSU.CRLF.7-3-4.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Hesan.He3
+PackageVersion: 1.3.7
+PackageLocale: en-US
+Publisher: Shenzhen Hesan Technology Co., Ltd.
+PublisherUrl: https://he3.app/en/
+PublisherSupportUrl: https://community.he3.app/
+PrivacyUrl: https://he3.app/en/pages/privacy/
```
Comment (stephengillie) [inline manifests/h/Hesan/He3/1.3.7/Hesan.He3.locale.en-US.yaml:10]:
```suggestion
PrivacyUrl: https://he3.app/en/privacy/
```

File: manifests/h/Hesan/He3/1.3.7/Hesan.He3.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+Author: Shenzhen Hesan Technology Co., Ltd.
+PackageName: He3
+PackageUrl: https://he3.app/en/
+License: Freeware
+LicenseUrl: https://he3.app/en/pages/terms/
```
Comment (stephengillie) [inline manifests/h/Hesan/He3/1.3.7/Hesan.He3.locale.en-US.yaml:15]:
```suggestion
LicenseUrl: https://he3.app/en/terms/
```

File: manifests/h/Hesan/He3/1.3.7/Hesan.He3.locale.zh-CN.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+PackageLocale: zh-CN
+Publisher: Shenzhen Hesan Technology Co., Ltd.
+PublisherUrl: https://he3.app/zh/
+PublisherSupportUrl: https://community.he3.app/
+PrivacyUrl: https://he3.app/zh/pages/privacy/
+Author: 深圳市氦三科技有限责任公司
+PackageName: He3
+PackageUrl: https://he3.app/zh/
+License: 免费软件
+LicenseUrl: https://he3.app/zh/pages/terms/
```
Comment (stephengillie) [inline manifests/h/Hesan/He3/1.3.7/Hesan.He3.locale.zh-CN.yaml:15]:
```suggestion
LicenseUrl: https://he3.app/zh/terms/
```

File: manifests/h/Hesan/He3/1.3.7/Hesan.He3.locale.zh-CN.yaml [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# Created with YamlCreate.ps1 v2.2.4 $debug=QUSU.CRLF.7-3-4.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.4.0.schema.json
+
+PackageIdentifier: Hesan.He3
+PackageVersion: 1.3.7
+PackageLocale: zh-CN
+Publisher: Shenzhen Hesan Technology Co., Ltd.
+PublisherUrl: https://he3.app/zh/
+PublisherSupportUrl: https://community.he3.app/
+PrivacyUrl: https://he3.app/zh/pages/privacy/
```
Comment (stephengillie) [inline manifests/h/Hesan/He3/1.3.7/Hesan.He3.locale.zh-CN.yaml:10]:
```suggestion
PrivacyUrl: https://he3.app/zh/privacy/
```

**Review by Trenly — APPROVED** [review]

**Pull request merged**